blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java

```diff
@@ -4,10 +4,7 @@
 import hudson.ExtensionList;
 import hudson.FilePath;
 import hudson.console.AnnotatedLargeText;
-import hudson.model.Action;
-import hudson.model.FileParameterValue;
-import hudson.model.ParameterDefinition;
-import hudson.model.ParameterValue;
+import hudson.model.*;
```

Comment:
this should be specific imports, not wildcard

Reply:
(wildcard is ok for test code but this is prod code)

Reply:
There's a setting in Idea that will do this for you when you optimise imports.

Reply:
@sophistifunk I had a problem with the latest intelliJ and optimise imports wasn't doing anything, I downgraded to the previous major version and it did the trick, thanks